source/weeknotes/index.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+layout: singlepage
+blog: weeknotes
+---
+
+# Weeknotes
+
+We send out an update on our work each week. Here’s the latest, and there’s
+a <a href="#previous">list of the older ones below</a>.
+
+---
+
+<h2>Latest: <%= blog.articles.first.title %></h2>
```

Comment:
Don't think the "Latest:" is that useful

Reply:
Even more radical: I don't think the "Weeknotes" `h1` and following paragraph is useful either. Just go straight into the latest weeknote as an `h1` ( as you've done on `/blog/` )

Reply:
Suggested in https://github.com/fluidkeys/www.fluidkeys.com/pull/46

Reply:
Sounds good! Simpler that way too!